src/t8_vtk/t8_vtk_write_ASCII.cxx

```diff
@@ -327,25 +327,40 @@ t8_forest_vtk_cells_elementid_kernel (t8_forest_t forest, [[maybe_unused]] const
   return 1;
 }
 
+// TODO: Move to forest_general.h
+static t8_locidx_t
```

Comment:
Please document the parameters of the function.

Reply:
Done